Sources/Playdate/Documentation.docc/BuildingTheExamples.md

```diff
@@ -88,4 +91,5 @@ Select your Swift nightly toolchain from the Xcode > Toolchains menu item.
 
 Select the Run icon in the toolbar or press Cmd+R on your keyboard to start a build. This will first build the example for the host machine using `xcbuild`. After a successful initial build, Xcode will automatically build again with `make` then open the Simulator with the newly built game.
 
-> Note: Learn more about [Xcode](https://developer.apple.com/xcode/)
+> ![Note]
```

Comment:
New GFM syntax https://github.com/orgs/community/discussions/16925

Widely used in Swift projects recently:
eg. https://github.com/apple/swift-testing/blob/b05678cacd825d531f1e0e6c167d6869c7961bc2/README.md?plain=1#L22-L29

Reply:
swift-docc -> swift-markdown -> cmark(GFM)

Looks like we have not updated our cmark(GFM) version to support.

So we'd better use it at most in README.md currently.

I'll revert to use the old syntax then.